### ISSUE TITLE
PageDataManager.instance().getRecords returns Duplicate records - 1.0.5

### ISSUE THREAD

**Issue by shivajibr** [xframium/xframium-java#45]
The following PageDataManager API is returning double the records available. 

PageData[] testSuiteList = PageDataManager.instance().getRecords("TestSuite"); 

If there is one record in the excel data sheet, the method returns 2 records, one of them being duplicate

I see there is a change in the addRecord functionality in 1.0.5 from 1.0.4 which is probably the root cause. This method is most likely inserting 2 records

public void addRecord( PageData pageData )
    {
        allRecordMap.get( pageData.getType() ).addRecord( pageData );

```
    if ( !pageData.isActive() )
        return;

    Deque<PageData> dataList = recordMap.get( pageData.getType() ).getRecordList();
    idMap.put( pageData.getType() + "." + pageData.getName(), pageData );
    dataList.offer( pageData );
}
```

**Comment by shivajibr** [comment]
Same issue observed when using XML test files and Loop keyword.

This API is returning duplicate records in KWSLoop keyword
List<PageData> dataArray = rootRecord.getPageData( valueSet[ 1 ] ); // Line no 132

**Comment by xframium** [comment]
Resovled - fixed in 1.0.6

**Comment by ghost** [comment]
Hi is 1.0.6 available to be pulled yet?
